simplq/src/components/pages/Admin/TourSteps.jsx

```diff
@@ -62,11 +64,23 @@ export const getToursteps = (screenInnerWidth) => {
   ];
 };
 
-export const hasUserBeenOnTour = () => {
-  const tourStatus = localStorage.getItem('__user_been_on_tour__');
-  if (tourStatus === null) {
-    localStorage.setItem('__user_been_on_tour__', true);
-    return true;
+export const stepChange = (stepNumber) => {
```

Comment:
I was thinking the same of moving it to common/Tour/Tour.jsx, but since it has hard coded value for handling specifically for the admin screen, I skipped it. Ideally, it must be first and last step and stepChange has a single parameter ie. stepNumber. Let me check, how we can generically handle the first and last step.

Reply:
Maybe we can use `props.toursteps`'s length to determine the last step.